docs/new-architecture-appendix.md

```diff
@@ -38,9 +38,9 @@ This is recommended over using plain `Object`, for type safety.
 
 **Example:** `{| foo: string, ... |}`
 
-| Nullable Support?                             | Android (Java) | iOS |
-| --------------------------------------------- | -------------- | --- |
-| <code>?{&#124; foo: string, ...&#124;}</code> | -              | -   |
+| Nullable Support?                                  | Android (Java) | iOS |
+| -------------------------------------------------- | -------------- | --- |
+| <code>?&#123;&#124; foo: string, ...&#124;}</code> | -              | -   |
```

Comment:
I agree. Can we use the HTML code also for the `}`?

Reply:
Yes, we can do that. 
However, it is not mandatory because MDX v2 only fails due to the opening characters. 
If you think it would make this easier to understand, 👍

Reply:
Used `&#125;` instead of `}` for consistency